src/plugins/plot/inspector/forms/LegendForm.vue

```diff
@@ -54,7 +54,7 @@
                 <option value="nearestValue">Nearest value</option>
                 <option value="min">Minimum value</option>
                 <option value="max">Maximum value</option>
-                <option value="units">Units</option>
+                <option value="unit">Unit</option>
```

Comment:
@charlesh88 Did you want this to be "Units" or maybe "Unit(s)"? You had mentioned there could be more than one type?

Reply:
@jvigliotta @charlesh88 , I didn't see this conversation resolved, but I assume, from the review outcome, that my initial change (`Unit`) was accepted in this case.

Nevertheless, I understand @charlesh88's point of view, which makes sense to me too on a use case basis. Let me add a few comments just for closure on the topic:
- In lines 52 to 57, the option values are related to a single telemetry point or singleton as you mentioned.
- Regarding the values to display when the legend is expanded (https://github.com/nasa/openmct/blob/9e545f763afb19d2e96e8d75d91c7084b0947850/src/plugins/plot/inspector/forms/LegendForm.vue#L67-L93), the available options also relate to a single telemetry entry (`Nearest timestamp`, `Nearest value`, etc).
<br>
So, for all these cases, we would agree on the choice of `Unit`, and to be thorough, it would have been better to rename `showUnitsWhenExpanded` to `showUnitWhenExpanded`.